### ISSUE TITLE
[Accessibility][Headings and Labels - Property Pane_Adaptive dialog]: 'Learn More' link should be more descriptive.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2133]
**User Experience** 
When the user navigate on the page using tab and once the focus reached to 'Learn More' link users face problem understanding the purpose as there is no announcement that this learn more is related to which heading. This makes the task time consuming as user will have to click on 'Learn More' and read the content to relate with the meaning of learn more link. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to the "Bot name.main" available under filter edit field and activate it.
5. Navigate to 'Adaptive dialog' available at right side of the pane.
6. Navigate to 'Learn More' link and observe the issue.

**Actual:** 
User face difficulty for understanding the purpose of 'Learn More' link if navigate using tab key as there are multiple 'Learn More' link available on the page. As well as screen reader only announces the name of the link as "Learn More Link" only.

**Expected:**
The narrator should announce some descriptive information which justify the propose of the 'Learn More' link.
Ex- It should announce the related topic with learn more link as 'Adaptive dialog Learn more link".

**Note:**
Same Issue is observed for all the Property Pane which have similar control such as 'Property Pane_Ask a question (Other)'.

**MAS Impacted:** [MAS1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment:**
![MAS1 3 1_Learn more link should be more descriptive](https://user-images.githubusercontent.com/60913131/75559045-c8c9fa00-5a68-11ea-8242-5b595981cc24.png)

**Comment by ashish315** [comment]
@cwhitten As checked still learn more is not descriptive but UI has been changed now, we have only one learn more link.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by corinagum** [comment]
@tdurnford @a-b-r-o-w-n  This didn't get ported over for some reason.

**Comment by a-b-r-o-w-n** [comment]
@corinagum @tdurnford will fix this today I think.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
